lib/js/styles/Colors/Colors.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<div class="sb__presentation">
+		<div class="sb__row">
+			<ol class="sb__list">
+				<li v-for="(colors, colorCategory) in rawColors" :key="colorCategory" class="sb__legend">
+					{{ colorCategory }}
+				</li>
+			</ol>
+		</div>
+		<div v-for="(colors, colorCategory) in rawColors" :key="colorCategory" class="sb__row">
+			<h2>{{ colorCategory }}</h2>
+			<div v-for="color in colors" :key="color.id" class='sb__table'>
```

Comment:
class name is not matching with tag name. Could you find a better name?

Reply:
Maybe `sb_colorsMap`?

Reply:
for me, it looks like a color definition, so maybe `colorDefinition`?

BTW. where did the `sb` prefix come from?

Reply:
`sb` is from Storybook, I had no idea what prefix do we use in styles which are global in SB. This prefix is not necessary, classes `colorDefinition` `colorDefinition__id` `colorDefinition__value` looks ok @kvas-damian